fga-eps-mds/2020.1-stay-safe-secretary-service
# Add new filters on crime endpoint

## Issue
Resolves #32 


## Description

Implemented query filters for cities, for months and a header flag to get the crimes per 100k inhabitants and returned the crimes in decreasing order of quantities.
Decided an annual range to provide a classification (1 a 6) for each crime when the flag per_capita is passed.
Created a controller and tests for each secretary and changed crime route to 'api/crimes/'.
Updated swagger.

## Checklist:



## How to validate?

Run the API and make a get request to 'api/crimes/' with the implemented queries -> secretary (requested), nature, city, initial_month, final_month (initial_month and final_month must be used together) and with the per_capita header. All filters can be combined.
Check if controllers, tests, swagger documentation, per capita range, mongo queries, etc. are properly.
